public/api-docs/swagger.json

```diff
@@ -183,6 +183,41 @@
          }
         }
       }
+    },
+    "/trip/returntrip/:token":{
```

Comment:
Are you sure this route exists?

Reply:
Since you and @karamuka are working on similar stories, could you both use `/trip` as the route like @mifeille suggested? I don't think it's necessary to have `/one-way` or `/two-way` in the route. @muchristian should also use this route.

Reply:
yes 

> Since you and @karamuka are working on similar stories, could you both use `/trip` as the route like @mifeille suggested? I don't think it's necessary to have `/one-way` or `/two-way` in the route. @muchristian should also use this route.

yes we are using the same route that is /trips

Reply:
> Are you sure this route exists?

it's no longer there I refactored with Patrick's code